extra/2-piping.js

```diff
@@ -16,26 +16,35 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  return num1 + num2;
 }
 
-function multiply() {
-
+function multiply(num1, num2) {
+  return num1 * num2;
 }
 
-function format() {
-
+function format(str) {
+  return `£${str}`;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+
+// Because all the functions are nested and therefore difficult to read
+let badCode = format(multiply(add(startingValue, 10),2));
 
 /* BETTER PRACTICE */
 
-let goodCode =
+// Add the value `startingValue` with the number 10
+let goodCode = add(startingValue, 10);
```

Comment:
Hi Mahmoud,
I have reviewed 2-piping.js file. For me, I don't think reassign the variable with multi functions results is the good practice. but, by creating a variable for each function result till be more readable and easy to be edited.

let add_numbers = add(10, startingValue);
let multiply_numbers = multiply(add_numbers, 2);
let goodCode = format(multiply_numbers);

Keep the good work bro and don't make my comment bother you I am just like you don't know much.